### PR TITLE
Add react session replay snippets

### PR DESCRIPTION
**Pull request by tore-statsig**
Going to move these to github embeds next.  Also need to link off to the react docs (which dont exist yet for this sdk)

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://662947480115ae31a6845da8--cozy-fox-0defec.netlify.app

File: docs/session-replay/install.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+```jsx
+import { StatsigClient } from '@statsig/js-client';
+import { SessionReplay } from '@statsig/session-replay';
+
+const client = new StatsigClient(sdkKey,
+	{ userID: "some_user_id" },
+  { environment: { tier: "staging" } } // optional, pass options here if needed
+);
+await client.initializeAsync()
+const replayer = new SessionReplay(client);
```
Comment (daniel-statsig) [inline docs/session-replay/install.mdx:89]: we should use the `runStatsigSessionReplay` function

Comment (daniel-statsig) [reply docs/session-replay/install.mdx:89]: We also mention installing `@statsig/web-analytics` but we never use it anywhere. We should probably include a `runStatsigAutoCapture(client)` somewhere

Comment (tore-statsig) [reply docs/session-replay/install.mdx:89]: good call

**Pull request merged**